PendingReleaseNotes.md

```diff
@@ -39,6 +39,12 @@
     - Preexisting disks provisioned as `Filestore` OSDs will remain as `Filestore` OSDs
 - RGWs:
   - Ceph Object Gateway are automatically configured to not run on the same host if hostNetwork is activated
+- Device Filtering made configurable for the user by adding an environment variable, and also by keeping the default filters intact
```

Comment:
Looks like you had some extra notes remaining after resolving a merge conflict

Reply:
Sorry. Solved. Thx!